lib/creek/styles/converter.rb

```diff
@@ -45,16 +46,18 @@ def self.call(value, type, style, options = {})
         when 'b'
           value.to_i == 1
         when 'str'
-          value
+          convert_html_entities(value)
```

Comment:
Probably the right thing to do is applying the htmlentities also in case of shared string (now not considered by my commit
`

        when 's' # shared string
          convert_html_entities(options[:shared_strings][value.to_i])
        when 'n' # number
          value.to_f
        when 'b'
          value.to_i == 1
        when 'str'
          convert_html_entities(value)
        when 'inlineStr'
          convert_html_entities(value)
`